psl/schema-ast/src/parser/datamodel.pest

```diff
@@ -114,7 +114,7 @@ field_attribute = { "@" ~ path ~ arguments_list? }
 // ######################################
 arguments_list = { "(" ~ (argument ~ ("," ~ argument)*)? ~ trailing_comma? ~ ")" }
 argument = _{ named_argument | empty_argument | expression }
-empty_argument = { identifier ~ ":" }
```

Comment:
This allows us an empty argument span that includes all the whitespace until the next `,` or `)`.